kyt.config.js

```diff
@@ -10,8 +10,14 @@ module.exports = {
     // modify baseConfig as needed
     const jestConfig = Object.assign({}, baseConfig);
 
+    // When running the end-to-end tests change the testRegex value to match
+    // related tests
+    if (process.env.TEST_ENV && process.env.TEST_ENV === 'e2e') {
```

Comment:
We could also think about doing the same thing I do for production builds where I use a different kyt.config file for specific scenarios. Although this probably works fine as well.